gateway-server/src/main/java/org/apache/knox/gateway/websockets/GatewayWebsocketHandler.java

```diff
@@ -118,7 +123,17 @@ public Object createWebSocket(ServletUpgradeRequest req,
 
     try {
       final URI requestURI = req.getRequestURI();
-      final ClientEndpointConfig clientConfig = getClientEndpointConfig(req);
+      // get parameters for KNOXSSO service
```

Comment:
Is this part only needed for logging? Do we need that every time when createWebSocket is called, or this is was just left there for debugging purposes?